.github/workflows/deploy.yml

```
@@ -0,0 +1,112 @@
+      - uses: actions/checkout@v4
+
+      - name: Get submodules
+        run: |
+          git config --global url."https://".insteadOf git://
+          git submodule update --init --recursive
+
+      - name: Setup .NET 8
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: 8.0.x
+          include-prerelease: false
```

actionlint

input "include-prerelease" is not defined in action "actions/setup-dotnet@v4". available inputs are "cache", "cache-dependency-path", "config-file", "dotnet-quality", "dotnet-version", "global-json-file", "owner", "source-url"

(action)